data/35e/wizards_of_the_coast/core/dungeon_masters_guide/dmg_equip.lst

```diff
@@ -1516,7 +1517,7 @@ Staff (Illumination)		OUTPUTNAME:Staff of [NAME]								TYPE:Magic.Staff
+Staff (Power)			OUTPUTNAME:Staff of [NAME]		PROFICIENCY:WEAPON|Quarterstaff	TYPE:Magic.Staff.Weapon.Simple.Melee.Bludgeoning.Standard.Double.Quarterstaff	ALTTYPE:Weapon.Melee.Bludgeoning	COST:211000	WT:4		BASEITEM:Staff		CRITMULT:x2	CRITRANGE:1	ALTCRITMULT:x2	ALTCRITRANGE:1	DAMAGE:1d6	ALTDAMAGE:1d6	EQMOD:CHARGED_ITEM_50|CHARGES[50]	WIELD:TwoHanded	SIZE:M	SOURCEPAGE:p.245	BONUS:COMBAT|AC|2|TYPE=Luck	BONUS:WEAPON|DAMAGE,TOHIT|2|TYPE=Enhancement	SPROP:The staff of power is a very potent magic item, with offensive and defensive abilities. It is usually topped with a glistening gem, its shaft straight and smooth. It has the following powers: . Magic missile (1 charge) . Ray of enfeeblement (heightened to 5th level) (1 charge) . Continual flame (1 charge) . Levitate (1 charge) . Lightning bolt (heightened to 5th level) (1 charge) . Fireball (heightened to 5th level) (1 charge) . Cone of cold (2 charges) . Hold monster (2 charges) . Wall of force (in a 10-ft.-diameter hemisphere around the caster only) (2 charges) . Globe of invulnerability (2 charges) The wielder of a staff of power gains a +2 luck bonus to AC and saving throws. The staff is also a +2 quarterstaff, and its wielder may use it to smite opponents. If 1 charge is expended (as a free action), the staff causes double damage (*3 on a critical hit) for 1 round. A staff of power can be used for a retributive strike, requiring it to be broken by its wielder. (If this breaking of the staff is purposeful and declared by the wielder, it can be performed as a standard action that does not require the wielder to make a Strength check.) All charges currently in the staff are instantly released in a 30-footradius. All within 2 squares of the broken staff take points of damage equal to 8 * the number of charges in the staff, those 3 or 4 squares away take 6 * the number of charges in damage, and those 5 or 6 squares distant take 4 * the number of charges in damage. All those affected can make DC 17 Reflex saves to reduce the damage by half. The character breaking the staff has a 50% chance of traveling to another plane of existence, but if he does not, the explosive release of spell energy destroys him. Only certain items, including the staff of the magi (page 280) and the staff of power, are capable of being used for a retributive strike. After all charges are used up from the staff, it remains a +2 quarterstaff. (Once empty of charges, it cannot be used for a retributive strike.) Strong varied; CL 15th	BONUS:SAVE|Fortitude,Reflex,Will|2|TYPE=Luck
```

Comment:
Escape % needs to be added. But if we are fixing the words '30-footradius' should be '30-foot radius'

Reply:
SPROP not working as documented, will update text further to correct.